package.json

```diff
@@ -125,9 +125,11 @@
     ],
     "nohoist": [
       "packages/web-components/webpack",
+      "packages/web-components/@microsoft/eslint-config-fast-dna",
       "packages/web-components/@storybook/**",
       "packages/web-components/typescript",
       "packages/web-components/typescript/**",
+      "packages/web-components/**/typescript",
```

Comment:
This very thoroughly prevents web-components' different versions of typescript from being hoisted.
- First line: don't hoist direct dep
- Second line: don't hoist deps of typescript (actually not needed since typescript has no deps)
- Third line: don't hoist nested deps on typescript
  - This is important because at least two web-components deps have deps on typescript: `@microsoft/api-extractor` (TS 3.7.2) and `@microsoft/eslint-config-fast-dna` (TS 3.9.7)